### DISCUSSION TITLE
官方交流群

### DISCUSSION THREAD

**Discussion by ZakaryCode** [NervJS/taro#8944] [官方公告]
目前 Taro 交流主要在微信群

* Taro 开发交流 1⃣️  群（已满）
* Taro 开发交流 2⃣️  群（已满）
* Taro 开发交流 3⃣️  群（已满）
* Taro 开发交流 4️⃣  群（已满）
* Taro 开发交流 5️⃣  群（已满）
* Taro 开发交流 6️⃣  群（已满）
* Taro 开发交流 7️⃣  群（已满）
* Taro 开发交流 8️⃣  群（已满）
* Taro 开发交流 9️⃣  群（已满）
* Taro 开发交流 🔟  群（已满）
* Taro 开发交流 1⃣️ 1⃣️  群（已满）
* Taro 开发交流 1⃣️ 2⃣️  群（已满）
* Taro 开发交流 1⃣️ 3⃣️  群（已满）
* Taro 开发交流 1️⃣ 4️⃣  群（已满）
* Taro 开发交流 1️⃣ 5⃣️  群（已满）
* Taro 开发交流 1️⃣ 6️⃣  群（已满）
* Taro 开发交流 1️⃣ 7️⃣  群（已满）
* Taro 开发交流 1️⃣ 8️⃣  群（已满）
* Taro 开发交流 1️⃣ 9️⃣  群（已满）
* Taro 开发交流 2⃣️ 0⃣  群（已满）
* Taro 开发交流 2⃣️ 1⃣️  群（已满）
* Taro 开发交流 2⃣️ 2⃣️  群（已满）
* Taro 开发交流 2⃣️ 3⃣️  群（已满）
* Taro 开发交流 2⃣️ 4️⃣  群（已满）
* Taro 开发交流 2⃣️ 5⃣️  群（已满）
* Taro 开发交流 2⃣️ 6️⃣  群（已满）
* Taro 开发交流 2⃣️ 7️⃣  群（已满）
* Taro 开发交流 2⃣️ 8️⃣  群（已满）
* Taro 开发交流 2⃣️ 9️⃣  群（已满）
* Taro 开发交流 3⃣️ 0⃣  群（已满）
* Taro 开发交流 3⃣️ 1⃣️  群（已满）
* Taro 开发交流 3⃣️ 2⃣️  群（已满）
* Taro 开发交流 3⃣️ 3⃣️  群（已满）
* Taro 开发交流 3⃣️ 4️⃣  群（已满）
* Taro 开发交流 3⃣️ 5⃣️  群（已满）
* Taro 开发交流 3⃣️ 6️⃣  群（已满）
* Taro 开发交流 3⃣️ 7️⃣  群（已满）
* Taro 开发交流 3⃣️ 8️⃣  群（已满）
* Taro 开发交流 3⃣️ 9️⃣  群（已满）
* Taro 开发交流 4️⃣ 0⃣  群（已满）
* Taro 开发交流 4️⃣ 1⃣️  群（已满）
* Taro 开发交流 4️⃣ 2⃣️  群（已满）
* Taro 开发交流群 43（已满）
* Taro 开发交流群 44（已满）
* Taro 开发交流群 45（已满）
* Taro 开发交流群 46（已满）
* Taro 开发交流群 47（等待加入～）

![Taro](http://storage.360buyimg.com/taro-jd-com/static/contact_taro_qr.png?t=1)

其他交流群

- Taro RN
  - Taro RN核心交流群（已满）
  - Taro RN核心交流 2⃣️  群（已满）
  - Taro RN核心交流群 3（已满）
  - Taro RN核心交流群 4（等待加入～）

![Taro RN](http://storage.360buyimg.com/taro-jd-com/static/contact_taro_rn_qr.png?t=1)

- Taro Vue
  - Taro Vue 开发交流 1⃣️  群（已满）
  - Taro Vue 开发交流 2⃣️  群（已满）
  - Taro Vue 开发交流群 3（已满）
  - Taro Vue 开发交流群 4（等待加入～）

![Taro Vue](http://storage.360buyimg.com/taro-jd-com/static/contact_taro_vue_qr.png?t=1)

- Taro Harmony
  - Taro x Harmony 开发交流群 1 群（已满）
  - Taro x Harmony 开发交流群 2 群（等待加入～）

![Taro Harmony](http://storage.360buyimg.com/taro-jd-com/static/contact_taro_harmony_qr.png?t=1)

- Taro x 飞书小程序
  - Taro x 飞书小程序交流群（等待加入～）

![Taro Lark](http://storage.360buyimg.com/taro-jd-com/static/contact_taro_lark_qr.png?t=1)

- NutUI x Taro
  - NutUI x Taro 开发交流群（等待加入～）

![NutUI x Taro](http://storage.360buyimg.com/taro-jd-com/static/contact_taro_nutui_qr.png?t=1)

- Taro UI
  - TaroUI 开发交流群（已满）
  - TaroUI 开发交流群2（已满）
  - TaroUI 开发交流群3（等待加入～）

![Taro UI](http://storage.360buyimg.com/taro-jd-com/static/contact_taro_ui_qr.png?t=1)

**Comment by ZakaryCode** [comment]
欢迎大家加入～

**Reply by levi-Ma** [reply]
47群已满员

**Reply by louwenqiang** [reply]
还能有新群吗

**Reply by TheKonka** [reply]
你直接扫码打开的就是新群

**Reply by qianqianWXQ** [reply]
群又满了

**Reply by suyanqa** [reply]
群慢满了,开一下新群吧

**Comment by TheKonka** [comment]
球球了，能不能不要用微信

**Comment by xiaxiaoke** [comment]
还会有新群么

**Reply by TheKonka** [reply]
你直接扫码打开的就是新群吧

**Reply by xiaxiaoke** [reply]
提示群已满人

**Comment by HHeLiBeB-C** [comment]
难道群满了就后续不更新了？

**Reply by honlyHuang** [reply]
适配的哪个端的群？

**Reply by crise1993** [reply]
适配rn端的群，目前有卡点，需要咨询大佬～🙏

**Reply by crise1993** [reply]
> 适配的哪个端的群？

适配rn端的群，目前有卡点，需要咨询大佬～🙏

**Reply by lsycxyj** [reply]
rn还有群吗？从模板创建项目，一接taro-ui又各种构建报错（我只引了一个tabbar，它所依赖的taro-rn引用各种依赖又报错），自己不断打patch构建通过了，还是跑不起来：
rn：0.73.11
expo：50
目前卡在先是这个报错
```
Invariant Violation: TurboModuleRegistry.getEnforcing(...): 'RNCClipboard' could not be found. Verify that a module by this name is registered in the native binary.Bridgeless mode: false. TurboModule interop: false. Modules loaded: {"NativeModules":["PlatformConstants","LogBox","Timing","AppState","SourceCode","BlobModule","WebSocketModule","DevSettings","DevToolsSettingsManager","Networking","Appearance","DevLoadingView","HeadlessJsTaskSupport","UIManager","DeviceInfo","DeviceEventManager","RNGestureHandlerModule","I18nManager","RNLocalize","SoundManager","RNCSafeAreaContext","ImageLoader","IntentAndroid","NativeAnimatedModule","RNCAsyncStorage"],"TurboModules":[],"NotFound":["NativePerformanceCxx","NativePerformanceObserverCxx","RedBox","BugReporting","LinkingManager","RNSModule","FrameRateLogger","KeyboardObserver","PlatformLocalStorage","RNC_AsyncSQLiteDBStorage","RNCClipboard"]}, js engine: hermes
```
看github [issue](https://github.com/react-native-clipboard/clipboard/issues/246)像是人家有问题，我试着打patch改成对应的expo-clipboard，安卓构建日志：
```
Using expo modules
  - expo-application (5.8.3)
  - expo-av (13.10.6)
  - expo-clipboard (5.0.1)
  - expo-constants (15.4.6)
  - expo-device (5.9.4)
  - expo-file-system (16.0.9)
  - expo-font (11.10.3)
  - expo-keep-awake (12.8.2)
  - expo-modules-core (1.11.14)
  - expo-modules-core$android-annotation (1.11.14)
  - expo-modules-core$android-annotation-processor (1.11.14)
```
但又报这个错
```
 ERROR  Error: Cannot find native module 'ExpoClipboard', js engine: hermes
    at AtTabBar (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.beamshare&modulesOnly=false&runModule=true:133553:36)
```
@honlyHuang @tutuxxx 大哥有没群指导一下

**Reply by ShaoGongBra** [reply]
用 duxapp 开发 RN，Taro 里面对 RN 支持最好的

**Comment by G-WangZH** [comment]
群满了都添加不上了

**Reply by honlyHuang** [reply]
适配的哪个端的群？

**Reply by TheKonka** [reply]
> 适配的哪个端的群？

立哥，扫第一个交流群的二维码就提示满人

**Reply by keenwon** [reply]
@honlyHuang 所有，所有二维码都提示「群已满人」

**Reply by crise1993** [reply]
适配rn端的群，目前有卡点，需要咨询大佬～🙏

**Reply by crise1993** [reply]
> 适配的哪个端的群？

适配rn端的群，目前有卡点，需要咨询大佬～🙏

**Reply by chenweiliang6** [reply]
> 适配的哪个端的群？

交流群还有吗，大哥

**Reply by ShaoGongBra** [reply]
> > 适配的哪个端的群？
> 
> 交流群还有吗，大哥

看看下面的duxapp群

**Comment by ShaoGongBra** [comment]
推荐个duxapp群，这是基于Taro的模块化开发框架，兼容开发小程序、APP、H5，鸿蒙
![wexin](https://github.com/user-attachments/assets/aed12a26-417e-4f6f-af3d-1f6e4bf8f170)

**Reply by feng943321546** [reply]
已经服了，脚手架初始化项目也能报错

**Reply by honlyHuang** [reply]
截图出来看看的
